modules/serverless-using-jobsink.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * serverless/eventing/event-sinks/serverless-jobsink.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-using-jobsink_{context}"]
+= Using JobSink
+
+When an event is sent to a `JobSink`, Eventing creates a `Job` and mounts the received event as JSON file at `/etc/jobsink-event/event`.
+
+.Procedure
+
+. Create a `JobSink` object definition as a YAML file:
++
+.JobSink YAML
+[source,yaml]
+----
+apiVersion: sinks.knative.dev/v1alpha1
+kind: JobSink
+metadata:
+  name: job-sink-logger
+spec:
+  job:
+    spec:
+      completions: 1
+      parallelism: 1
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: main
+              image: docker.io/library/bash:5
+              command: [ "cat" ]
+              args:
+                - "/etc/jobsink-event/event"
+----
+
+. Apply the `JobSink` YAML file:
++
+[source,terminal]
+----
+$ oc apply -f <job-sink-file.yaml>
+----
+
+. Verify `JobSink` is ready:
++
+[source,terminal]
+----
+$ oc get jobsinks.sinks.knative.dev
+----
+Example output:
++
+[source,terminal]
+----
+NAME              URL                                                                          AGE   READY   REASON
+job-sink-logger   http://job-sink.knative-eventing.svc.cluster.local/default/job-sink-logger   5s    True
+----
+
+. Trigger a `JobSink`. `JobSink` can be triggered by any xref:../event-sources/knative-event-sources.adoc#knative-event-sources[event source] or xref:../triggers/serverless-triggers.adoc#serverless-triggers[trigger].
```

Comment:
We do not mention xrefs in module. You can mention this link in the assembly file in a new additional resources section.